nuxt.config.ts

```diff
@@ -9,6 +9,7 @@ export default defineNuxtConfig({
       "process.env.DEBUG": false,
     },
   },
+  ssr: false,
```

Comment:
Also, need to find out why this is causing a bug for you and remove this, as SSR is important to Nuxt